### DISCUSSION TITLE
FA5 Pro Compatability

### DISCUSSION THREAD

**Discussion by vidoardes** [fomantic/Fomantic-UI#1234] [General] [answered]
The fact that this project uses FA5 is awesome, but it is actually quite difficult to use FA5 Pro alongside it if you have a license. It would be nice to get all of the benefits of using the `icon` class (sizing, spacing, collapsing etc.) but instead of using the built in FA5 supply your own link to FA5 Pro.

Combining the two like below often has issues, especially with the duotone icons.

`<i class="fad fa-egg icon">`

**Comment by y0hami** [comment, marked as answer]
@vidoardes We offer a tool to generate the required files to use FA5 Pro in fomantic just like how FA5 Free works 🎉 Take a look https://github.com/fomantic/create-fomantic-icons

**Comment by vidoardes** [comment]
Looks good, I am using the CDN version of Fomatic at the moment, might have to switch to npm
